pkg/model/manifest.go

```diff
@@ -1020,6 +1021,70 @@ func (d *Dependency) GetTimeout(defaultTimeout time.Duration) time.Duration {
 	return defaultTimeout
 }
 
+// ExpandVarsInSection sets dependencies values if values fits with list params
+func (d *Dependency) ExpandVarsInSection(envKeyPairs []string) error {
```

Comment:
nit: `ExpandVars` what does the `InSection` mean?

Reply:
`InSection` means "in dependencies section" but I don't think at this point is needed to mention it. I think is enough to use `ExpandVars`, being called by `Dependency` struct I think it gives the needed context to understand what the func does. Done at b7009a55146dff577014105f540e9c74c125a621 😄